zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterEvent.java

```diff
@@ -1,22 +1,5 @@
 /**
- * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It's strange because files that you mentioned is generated by `thrift` automatically.